src/sentry/static/sentry/app/components/stream/group.jsx

```diff
@@ -158,9 +327,50 @@ const GroupCheckbox = styled(Box)`
   }
 `;
 
-const StyledCount = styled(Count)`
-  font-size: 18px;
-  color: ${p => p.theme.gray600};
+const PrimaryCount = styled(Count)`
+  font-size: ${p => p.theme.fontSizeExtraLarge};
+  color: ${p => p.theme.gray700};
+`;
+
+const SecondaryCount = styled(({value, ...p}) => <Count {...p} value={value} />)`
+  font-size: ${p => p.theme.fontSizeExtraLarge};
+  color: ${p => p.theme.gray500};
+
+  :before {
+    content: '/';
+  }
+`;
+
+const TooltipContent = styled(({children, ...p}) => (
+  <table {...p}>
+    <tbody>{children}</tbody>
+  </table>
+))`
+  margin: 0;
+`;
+
+const TooltipRow = styled('tr')`
+  padding: ${space(0.5)} ${space(1)};
+  display: block;
+`;
+
+const TooltipText = styled('td')`
+  padding-left: ${space(1.5)};
+  font-weight: normal;
+`;
+
+const TooltipCount = styled('td')`
+  font-weight: bold;
+`;
+
+const StyledIconTelescope = styled(({to, ...p}) => (
+  <td {...p}>
+    <Link title={t('Open in Discover')} to={to}>
```

Comment:
How does the tooltip inside a tooltip look?

Reply:
They are the blue telescope icons in the screenshots